### ISSUE TITLE
React Table throws JS errors

### ISSUE THREAD

**Issue by Djfaucette** [patternfly/patternfly-react#2656]
In a default testing setup inside a React app, I'm getting an error in react/table:

```
./node_modules/lodash-es/lodash.js:10
    export { default as add } from './add.js';
    ^^^^^^

    SyntaxError: Unexpected token export
```

I'm using @patternfly/react-table v2.14.23.

**Comment by priley86** [comment]
hi @Djfaucette - is your environment using Babel/es6 constructs?

I would also double check that you also have a default export inside of `add.js`, ie:
```
function add () {}
export default add
```
https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export

I don't know that this is anything specific to table. We'll need more info here 😄

**Comment by Djfaucette** [comment]
Hi, Patrick

Thanks for the quick reply. The error is being thrown from: `./node_modules/@patternfly/react-table/dist/js/components/Table/base/evaluate-transforms.js`

So my guess is that there is an imported file that has a strange export statement somewhere in that component.

**Comment by Djfaucette** [comment]
I just noticed I didn't include the full file paths in my issue. Whoops!

```./node_modules/lodash-es/lodash.js:10
    export { default as add } from './add.js';
    ^^^^^^

    SyntaxError: Unexpected token export```

That's the original error I am seeing (filepath added) and the file throwing the error is the `/evaluate-transforms.js` above.

**Comment by priley86** [comment]
This seems build related in your environment if ESM is not recognized. We use ESM is many apps now downstream along w/ webpack for tree shaking dependencies, however we do still build CommonJS and UMD modules (you can see those in the `dist/js` and `dist/umd` folders of our npm packages.  We recently switched from `lodash-es` to `lodash` - maybe updating can help you? FWIW, I've tested latest alpha packages and cannot reproduce this in my downstream ESM apps:

```
    "@patternfly/patternfly": "2.23.0",
    "@patternfly/react-charts": "4.7.4",
    "@patternfly/react-core": "3.82.3",
    "@patternfly/react-table": "2.16.7",
    "@patternfly/react-tokens": "2.6.14",
    "@patternfly/react-topology": "2.7.16",
    "@patternfly/react-virtualized-extension": "1.1.101"
```

I would try updating and ensure you're targeting CommonJS modules if your environment does not support ESM.

Otherwise, you may have to share your repo/branch changes w/ us...

**Comment by rachael-phillips** [comment]
@Djfaucette did you resolve your issue here?

**Comment by Djfaucette** [comment]
@rachael-phillips Hi, Rachael

No, but I do have a little bit more context after heading down a rabbit hole.

This is an issue I encountered only after upgrading my version of react-table. This is an issue with my testing framework, Jest, not being able to resolve the export syntax in a testing environment. I initially thought it was a React error since it was showing in a local dev environment. My application did compile, but I decided to go a different direction since I couldn't resolve the error in my testing suite.
